server/setup.py

```diff
@@ -51,9 +51,9 @@
     ],
     extras_require={
         'onnx': [
-            'onnxruntime',
             'onnx',
             'onnxmltools',
+            'onnxruntime',
```

Comment:
As you mentioned, we can always install `onnxruntime-gpu`?

Reply:
I've tested on x86 machines with/without GPU, both work fine. The CI failed for unknown reasons